# this message type keeps the four cubic coefficients and intial and fine time
float64 hand_height
float64 gesture_rate
string fluency
